mozilla-release/browser/components/migration/FirefoxProfileMigrator.js

```diff
@@ -180,7 +241,262 @@ FirefoxProfileMigrator.prototype._getResourcesInternal = function(sourceProfileD
+          let db = yield Sqlite.openConnection({
+            path: cookiesFile.path
+          });
+
+          try {
+            let rows = yield db.execute(`SELECT name, value,
+                                                host, path,
+                                                expiry, isSecure,
+                                                isHttpOnly
+                                          FROM moz_cookies`);
+            for(let row of rows) {
+              Services.cookies.add(row.getResultByName("host"),
+                                   row.getResultByName("path"),
+                                   row.getResultByName("name"),
+                                   row.getResultByName("value"),
+                                   row.getResultByName("isSecure"),
+                                   row.getResultByName("isHttpOnly"),
+                                   false,
+                                   row.getResultByName("expiry"),
+                                   {});
+            }
+          } finally {
+            yield db.close();
+          }
+        }).then(() => aCallback(true),
+        e => { Cu.reportError(e); aCallback(false) });
+      }
+    };          
+  }.bind(this);
+
+  let getFormDataResource = function(aFileName) {
+    let formDataFile = this._getFileObject(sourceProfileDir, aFileName);
+    if (!formDataFile.exists())
+      return null;
+
+    return {
+      type: MigrationUtils.resourceTypes.FORMDATA,
+
+      migrate(aCallback) {
+        return Task.spawn(function* () {
+          let db = yield Sqlite.openConnection({
+            path: formDataFile.path
+          });
+
+          try {
+            let rows = yield db.execute(`SELECT fieldname,
+                                                value,
+                                                timesUsed,
+                                                firstUsed,
+                                                lastUsed
+                                        FROM moz_formhistory`);
+            let changes = [];
+            for(let row of rows) {
+              changes.push({
+                            op: "add",
+                            fieldname: row.getResultByName("fieldname"),
+                            value:     row.getResultByName("value"),
+                            timesUsed: row.getResultByName("timesUsed"),
+                            firstUsed: row.getResultByName("firstUsed"),
+                            lastUsed:  row.getResultByName("lastUsed"),
+                          });
+            }
+            FormHistory.update(changes);
+          } finally {
+            yield db.close();
+          }
+        }).then(() => aCallback(true),
+        e => { Cu.reportError(e); aCallback(false) });
+      }
+    };          
+  }.bind(this);
+
   let types = MigrationUtils.resourceTypes;
+  if (!this.startupOnlyMigrator && !MigrationUtils.isStartupMigration) {
```

Comment:
Yes. You're right. It's always `true` because we define it here:
```
Object.defineProperty(FirefoxProfileMigrator.prototype, "startupOnlyMigrator", {
   get: () => false
});
```
So I left it like that in case we want to disable this migration option, we just need to change the `get: () => true`